mrpt_bridge/CHANGELOG.rst

```diff
@@ -2,6 +2,18 @@
 Changelog for package mrpt_bridge
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+0.1.19 (2017-08-25)
+-------------------
+* Added mrpt_ros bridge for the following messages
+* mrpt::obs::CObservationImage <--> sensor_msgs/Image
+* mrpt::obs::CObservationIMU <--> sensor_msgs/IMU
+* mrpt::obs::CObservationGPS <--> sensor_msgs/NavSatFix
+* mrpt::obs::CObservationRange <--> sensor_msgs/Range
+* mrpt::obs::CObservationStereoImages <--> 3xsensor_Image for left, right and disparity
+* also added the file test_Bridge to test the above conversions and publishing on respective ROS topics
+* Contributors: Raghavender Sahdev
+
```

Comment:
Thanks for updating the CHANGELOG! 

However, normally this file is updated automatically as part of using the automated "bloom" scripts from ROS. 
But your content will be useful to assist in the cleanup of the automatically-generated changelog, when time comes for such a new release! ;-)